cli/functionary/client.py

```diff
@@ -0,0 +1,98 @@
+import json
+
+import click
+import requests
+
+from .config import get_config_value
+
+
+def get(endpoint):
+    """
+    Gets any data associated with an endpoint from the api
+
+    Args:
+        Endpoint: the name of the endpoint to get data from
+
+    Returns:
+        Data: Data from endpoint as Python list/dict
+
+    Raises:
+        ClickException if failed connection, timeout, or could not retrieve endpoint
+
+    """
+    token = get_config_value("token")
+    header = {"Authorization": f"Token {token}"}
+    url = get_config_value("host") + f"/api/v1/{endpoint}"
+
+    try:
+        response = requests.get(url, headers=header)
+    except requests.ConnectionError:
+        raise click.ClickException("Could not connect to host")
+    except requests.Timeout:
+        raise click.ClickException("Timeout occured while trying to connect to host")
+
+    if response.ok:
+        data = json.loads(response.text).get("results")
+        return data
+    else:
+        if response.status_code == 401:
+            raise click.ClickException("Unauthorized request, try logging in again.")
+        elif response.status_code == 403:
+            raise click.ClickException("Request forbidden")
```

Comment:
Let's try to be more clear in the messages returned.  Maybe:

"You do not have access to perform this action"